app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+class CommentsController < ApplicationController
+  before_action :set_comment, only: %i[edit update destroy]
+
+  def create
+    @comment = @commentable.comments.build(comment_params)
+    @comment.user_id = current_user.id
```

Comment:
細かいですが、関連が張ってあればこちらでも同じように動作します〜

```suggestion
    @comment.user = current_user
```

個人的にはこちらの書き方をよく見ます。いわゆる Rails っぽい書き方かなと思います

Reply:
なるほどです！
コメントとユーザー間にリレーションがあれば、`user`=`user_id`（プライマリーキー）ということですね！！
少し混乱していた部分だったので、勉強になりました💪

Reply:
いえ、こちらは Comment のインスタンスメソッド `user=` を呼び出して、@comment に user を関連づけているという形ですね。「メソッド `user=` なんて追加した覚えはないよ！」と思うかもですが、実は Comment モデルに書いた `belongs_to`メソッドによって追加されています。
この辺りの仕組みについては https://railsguides.jp/association_basics.html#belongs-to%E9%96%A2%E9%80%A3%E4%BB%98%E3%81%91 を読んでみてください〜

Reply:
`belongs_to`で関連付けしたものに対しても、そのモデルのカラムをメソッドとして使えるようになるという部分クリアになりました😆
ありがとうございます！！